contrib/mergeBenchmarkSets.py

```diff
@@ -98,7 +98,7 @@ def get_witness_result(witness, verification_result):
     if status_from_validation == status_from_verification:
         return status_from_verification, category_from_verification
     # An invalid witness counts as error of the verifier.
-    if status_from_validation == "ERROR (invalid witness syntax)":
+    if status_from_validation.startswith("ERROR"):
```

Comment:
Please note my previous comment:

I think it is possible that a validator returns `ERROR` due to an internal error (e.g., because it cannot parse the program). In this case the witness shouldn't be labeled as invalid, right?

Reply:
Also note that the invalid witness syntax really should only be decided by the linter, not by some individual validator. If a validator falsely claims that the witness syntax is invalid, but all other validators do not do this, how do we detect/handle this? Especially cases where the linter says the witness is alright but a validator claims the syntax is invalid. Currently we have the following ensured by the current state of the tool-info modules:
```
benchexec/benchexec/tools $ grep invalid -rHn
fshell-witness2test.py:106:            status += " (invalid witness file)"
smack.py:85:            if run.output.any_line_contains("invalid pointer dereference"):
smack.py:87:            elif run.output.any_line_contains("invalid memory deallocation"):
ultimate.py:448:                    status += " (invalid witness file)"
cbmc.py:95:                # in this case an exception is expected as the XML is invalid
cpachecker.py:160:                    status += " (invalid witness file)"
cpachecker.py:173:                    status += " (invalid witness file)"
impara.py:88:                    # in this case an exception is expected as the XML is invalid
theta.py:64:                status = "ERROR (invalid parameter)"
witnesslint.py:44:            return result.RESULT_ERROR + " (invalid witness syntax)"
```

So only the witness linter is allowed to produce "invalid witness syntax", so we will never see this from another tool. Other validators will at most say "invalid witness file". This is an indirect way to filter for the linter's result only. While this is not ideal, I think it works for now. We need to take care though to ensure this in variant still holds after our changes, and people need to be aware about this (maybe add a comment that the linter is the only one to be expected to return this verdict).